src/graph.rs

```diff
@@ -4483,6 +4507,54 @@ mod tests {
     }));
   }
 
+  #[test]
+  fn test_jsr_import_format() {
+    assert!(
+      validate_jsr_specifier(&Url::parse("jsr:@scope/mod@tag").unwrap())
+        .is_err(),
+      "jsr import specifier with tag should be an error"
+    );
+
+    assert!(
+      validate_jsr_specifier(&Url::parse("jsr:@scope/mod@").unwrap()).is_err()
+    );
+
+    assert!(validate_jsr_specifier(
+      &Url::parse("jsr:@scope/mod@1.2.3").unwrap()
+    )
+    .is_ok());
+
+    assert!(
+      validate_jsr_specifier(&Url::parse("jsr:@scope/mod").unwrap()).is_ok()
+    );
+  }
+
+  // TODO: Write a test that would demonstrate that jsr does not support version tags
+  //
```

Comment:
It would be great to have a more involved test or even an integration test. Suggestions welcome.

Reply:
I've added a test